http/handler.go

```diff
@@ -352,12 +352,67 @@ func handleUIStub() http.Handler {
 	stubHTML := `
 	<!DOCTYPE html>
 	<html>
-	<p>Vault UI is not available in this binary. To get Vault UI do one of the following:</p>
+	<style>
+	body {
+	color: #1F2124;
+	font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+	}
+
+	.wrapper {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+  height: 500px;
```

Comment:
Not sure if we care much about indenting, but do we want this in line with the others?